src/WebJobs.Extensions.ApiHub/WebJobs.Extensions.ApiHub.csproj

```diff
@@ -0,0 +1,116 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{091041AE-378D-42B0-AA0A-6A1F47C0BE2D}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>Microsoft.Azure.WebJobs.Extensions.ApiHub</RootNamespace>
+    <AssemblyName>Microsoft.Azure.WebJobs.Extensions.ApiHub</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
You need to enable code analysis (fxcop/stylecop) like all the other projects. See https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions.SendGrid/WebJobs.Extensions.SendGrid.csproj#L14